### PR TITLE
Book controller filter

### PR DESCRIPTION
**Pull request by maxy96**
# Summary
* Rack-reducer installed 
   * added in the index path to filter by author, genre and title

# Trello Card
https://trello.com/c/Xi5uXHUW/50-agregar-filtros-al-index-de-books

# Screenshot
Captura de Pantalla 2021-06-10 a la(s) 11 26 11

### CODE REVIEW

File: app/reducers/book_reducer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class BookReducer < Rack::Reducer
+    def initialize()
+        super(Book.all, 
+            -> (author:) { where('lower(author) like ?', "%#{author.downcase}%") },
+            -> (genre:) {where('lower(genre) like ?', "%#{genre.downcase}%")}, 
+            -> (title:) {where('lower(title) like ?', "%#{title.downcase}%")} 
+        )
+    end
+end
```
Comment (blacksam07) [inline app/reducers/book_reducer.rb]: Add newline at end of file

**Review by blacksam07 — APPROVED** [review]
👍🏽

File: app/reducers/book_reducer.rb (hunk 1 above)
Comment (maxy96) [reply app/reducers/book_reducer.rb]: a line break?

Comment (blacksam07) [reply app/reducers/book_reducer.rb]: All files should include an empty line at end of the file

Comment (maxy96) [reply app/reducers/book_reducer.rb]: ok!

**Review by blacksam07 — APPROVED** [review]

**Pull request merged**